cmake/external_llvm.cmake

```diff
@@ -91,6 +91,7 @@ ExternalProject_Add(
                 -DLIBOMP_OMPT_SUPPORT=OFF
                 -DCLANG_BUILD_TOOLS=OFF
                 -DCLANG_ENABLE_ARCMT=OFF
+                -DCLANG_ANALYZER_ENABLE_Z3_SOLVER=OFF
```

Comment:
Why is this change related to MLIR? This LLVM is the one used for CPU codegen.

Reply:
For CI we build LLVM only when MLIR is enabled. This flag is needed when building LLVM8 with clang9 or clang10.